packages/wrangler/src/deployment-bundle/entry.ts

```diff
@@ -70,8 +72,47 @@ export async function getEntry(
 					"For Pages, please run `wrangler pages dev` instead."
 			);
 		}
+
+		const compatibilityDateStr = [
+			new Date().getFullYear(),
+			(new Date().getMonth() + 1 + "").padStart(2, "0"),
+			(new Date().getDate() + "").padStart(2, "0"),
```

Comment:
Nit: you could create the date just once without doing it three times

Reply:
Also isn't using the current date as the compat date problematic (it generally shows a warning)? I thought that for that reason in C3 we use the lastest compat date that we fetch from npm? 🤔

Reply:
Yeah I realise that this is potentially tricky. But I was getting bogged down in scope creap.
Originally this just updating the text, then it was fixing up the package manager, then adding in name and compat date 😓 . I don't want to start re-implementing C3!
At the end of the day, this is just messaging, we are not automatically updating the user's code.
If the compat date issue is a problem, we could just leave a placeholder there - something like "<latest-compat-date>" but I felt like that was even more confusing and unhelpful.